frontend/talentsearch/src/js/components/login/LoginPage.tsx

```diff
@@ -17,7 +17,7 @@ const LoginPage: React.FC = () => {
   const intl = useIntl();
   const paths = useApiRoutes();
   const talentPaths = useTalentSearchRoutes();
-  const loginPath = paths.login(talentPaths.profile(), getLocale(intl));
+  const loginPath = paths.login(talentPaths.home(), getLocale(intl));
```

Comment:
We can't define a redirect to the users profile page here since we don't know their id at this point. The only solution I can think of is defining a 'me' profile route. Does anyone know another way of doing this? We have the same issue in register.